com.ibm.streamsx.topology/opt/python/packages/streamsx/scripts/streamtool.py

```diff
@@ -52,18 +52,37 @@ def _job_cancel(instance, job_id=None, job_name=None, force=False):
 ###########################################
 def _lsjobs_parser(subparsers):
     job_ls = subparsers.add_parser('lsjobs', help='List the jobs for a given instance')
+    job_ls.add_argument('--jobs', '-j', help='Specifies a list of job IDs.', metavar='job-id')
+    job_ls.add_argument('--users', '-u')
```

Comment:
Need help for `--users` and `--jobnames`. See if text can be copied from real stream tool.